src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -80,6 +83,95 @@ enum MiscSpells
     SPELL_MAGE_CHILLED                           = 205708
 };
 
+// 110909 - Alter Time
+class spell_mage_alter_time_aura : public AuraScript
+{
+    PrepareAuraScript(spell_mage_alter_time_aura);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
```

Comment:
This script doesn't need to override Validate at all, it is not directly using any of these spell ids